platforms/documentation/docs/src/docs/userguide/authoring-builds/other/validation_problems.adoc

```diff
@@ -237,6 +237,33 @@ Because there are no semantics associated with this combination, Gradle cannot d
 
 To fix this, you most likely need to remove the conflicting modifier annotation or check that the actual property type is what you intended.
 
+[[property_accessor_must_not_be_overridden]]
+== Property accessor must not be overridden
+
+This error indicates that a getter or a setter for a property annotated with `@ToBeReplacedByLazyProperty` has been overridden by user code.
```

Comment:
💭 Once we release the migration, this will need to be updated to say `@ReplacesEagerProperty`. We should notice, because we'll be removing `@ToBReplacedByLazyProperty`.